src/main/java/tds/content/mappers/ITSDocumentMapper.java

```diff
@@ -0,0 +1,92 @@
+/***************************************************************************************************
+ * Copyright 2017 Regents of the University of California. Licensed under the Educational
+ * Community License, Version 2.0 (the “license”); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the license at
+ *
+ * https://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required under applicable law or agreed to in writing, software distributed under the
+ * License is distributed in an “AS IS” BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for specific language governing permissions
+ * and limitations under the license.
+ **************************************************************************************************/
+
+package tds.content.mappers;
+
+import org.springframework.stereotype.Component;
+
+import tds.itemrenderer.data.ITSDocument;
+import tds.itemrenderer.data.ITSSoundCue;
+import tds.itemrenderer.data.ITSTutorial;
+import tds.itemrenderer.data.ITSTypes;
+import tds.itemrenderer.data.xml.itemrelease.ItemPassage;
+import tds.itemrenderer.data.xml.itemrelease.Itemrelease;
+import tds.itemrenderer.data.xml.itemrelease.Passage;
+
+@Component
+public class ITSDocumentMapper implements ItemDocumentMapper {
+    @Override
+    public void mapItemDocument(final ITSDocument document, final Itemrelease itemXml) {
+        document.setValidated(true);
+        if (itemXml.getVersion() != null) {
+            document.setVersion(Double.parseDouble(itemXml.getVersion().trim()));
+        } else {
+            document.setVersion(1.0);
+        }
+
+        if (document.getVersion() == 0) {
+            return;
+        }
+
+        if (itemXml.getItemPassage() != null) {
+            if (itemXml.getItemPassage().getClass() == Passage.class) {
+                document.setType(ITSTypes.ITSEntityType.Passage);
+            } else {
+                document.setType(ITSTypes.ITSEntityType.Item);
+            }
+        }
+
+        // make sure the document was defined as either an item or passage
+        if (document.getType() == ITSTypes.ITSEntityType.Unknown) {
```

Comment:
Can't this just be the else to the condition on line 41? The only way this will ever be true is if the item passage is null?